ex_functions/ex_roman_numerals.js

```diff
@@ -0,0 +1,88 @@
+const romanNumerals = [
+  { value: 1000, numeral: "M" },
+  { value: 900, numeral: "CM" },
+  { value: 500, numeral: "D" },
+  { value: 400, numeral: "CD" },
+  { value: 100, numeral: "C" },
+  { value: 90, numeral: "XC" },
+  { value: 50, numeral: "L" },
+  { value: 40, numeral: "XL" },
+  { value: 10, numeral: "X" },
+  { value: 9, numeral: "IX" },
+  { value: 5, numeral: "V" },
+  { value: 4, numeral: "IV" },
+  { value: 1, numeral: "I" },
+];
+
+convertToRomanNumerals = function (value) {
+  const lowerLimit = 1;
+  const upperLimit = 3999;
+  let output = "";
+
+  if (value < 1 || value > upperLimit) {
+    throw new Error(
+      `Invalid input ${value}. Please enter a number between ${lowerLimit} and ${upperLimit}.`
+    );
+  }
+
+  romanNumerals.forEach((numeral) => {
+    while (value >= numeral.value) {
+      output += numeral.numeral;
+      value -= numeral.value;
```

Comment:
Aquí estás modificando o parámetro de entra value e reutilizando un nome. que Según clean code debería pasar a unha variable auxiliar con un nome máis axeitado, por exemplo:
- output >> accumulated
- value >> remaining